profile.proto

```diff
@@ -41,6 +42,24 @@ message CreateProfileRequest {
     Location location = 13;
 }
 
+message Userfilters {
+    string name = 1;
+    Gender gender = 2;
+    FarmingType farmingType = 3;
+    int32 yearsSinceOrganicFarming = 4;
+    LandSizeInAcres landSizeInAcres = 5;
+}
+
+message GetAllProfilesRequest {
+    Userfilters Filters = 1;
+    int32 pageSize = 2;
+    int32 pageNumber = 3;
+}
+
+message GetAllProfilesResponse {
```

Comment:
This message is same as bultGetProfilesResponse, use one message don't declare duplicates. Give more generic name to this and use it in both rpc. (MultipleProfileResponse)